hostscripts/rpm-packaging/createproject.py

```diff
@@ -59,10 +59,13 @@ def upload_meta(project, build_repository, linkproject):
     description = ''
     if linkproject:
         if 'ZUUL_UUID' in os.environ:
-            description = """
+            description += """
 This project tests the following Zuul Change IDs: %(ZUUL_CHANGE_IDS)s\n
 Branch used: %(ZUUL_BRANCH)s\n
 Project used: %(ZUUL_PROJECT)s
+""" % (os.environ)
+        if 'BUILD_URL' in os.environ:
```

Comment:
imho we could/should just fold it into the if ZUUL_UUID condition above

Reply:
`BUILD_URL` is unrelated to zuul. I don't think we should mix that.

Reply:
it is being set by zuul builds :)

Reply:
It's not related to zuul. This is a jenkins think. Look eg. at https://ci.opensuse.org/job/docker-trigger-libyui-devel-latest/686/injectedEnvVars/ which has nothing to do with zuul but still has `BUILD_URL`.